easybuild/easyblocks/l/llvm.py

```diff
@@ -60,65 +271,938 @@ def __init__(self, *args, **kwargs):
+                # without resulting in relocation errors).
+                # See https://github.com/easybuilders/easybuild-easyblocks/pull/2799#issuecomment-1270621100
+                # Here, we add -Wno-unused-command-line-argument to CXXFLAGS to avoid these warnings alltogether
+                cflags = os.getenv('CFLAGS', '')
+                cxxflags = os.getenv('CXXFLAGS', '')
+                setvar('CFLAGS', "%s %s" % (cflags, '-Wno-unused-command-line-argument'))
+                setvar('CXXFLAGS', "%s %s" % (cxxflags, '-Wno-unused-command-line-argument'))
+
+            if self.full_llvm:
+                # See  https://github.com/llvm/llvm-project/issues/111667
+                to_add = '--unwindlib=none'
+                # for flags in ['CMAKE_C_FLAGS', 'CMAKE_CXX_FLAGS']:
+                for flags in ['CMAKE_EXE_LINKER_FLAGS']:
+                    ptr = self.runtimes_cmake_args[flags]
+                    if to_add not in ptr:
+                        ptr.append(to_add)
+
+                self._add_cmake_runtime_args()
+
+            # determine full path to clang/clang++ (which may be wrapper scripts in case of RPATH linking)
+            clang = which('clang')
+            clangxx = which('clang++')
+
+            self._cmakeopts['CMAKE_C_COMPILER'] = clang
+            self._cmakeopts['CMAKE_CXX_COMPILER'] = clangxx
+            self._cmakeopts['CMAKE_ASM_COMPILER'] = clang
+            self._cmakeopts['CMAKE_ASM_COMPILER_ID'] = 'Clang'
+
+            # Also runs of the intermediate step compilers should be made aware of the GCC installation
+            if LooseVersion(self.version) >= LooseVersion('19'):
+                self._set_gcc_prefix()
+                create_compiler_config_file(self.cfg_compilers, self.gcc_prefix, prev_dir)
+
+            self.add_cmake_opts()
+
+            change_dir(stage_dir)
+            self.log.debug("Configuring %s", stage_dir)
+            cmd = "cmake %s %s" % (self.cfg['configopts'], os.path.join(self.llvm_src_dir, 'llvm'))
+            run_shell_cmd(cmd)
+
+            self.log.debug("Building %s", stage_dir)
+            cmd = "make %s VERBOSE=1" % self.make_parallel_opts
+            run_shell_cmd(cmd)
+
+        change_dir(curdir)
+
+    def build_step(self, verbose=False, path=None):
+        """Build LLVM, and optionally build it using itself."""
+        if self.cfg['bootstrap']:
+            self.log.info("Building stage 1")
+            print_msg("Building stage 1/3")
+        else:
+            self.log.info("Building LLVM")
+            print_msg("Building stage 1/1")
+        # change_dir(self.llvm_obj_dir_stage1)
+        # super(EB_LLVM, self).build_step(verbose, path)
+        import shutil
+        change_dir(self.builddir)
+        print_msg("TESTING!!!: Copying from previosu build (REMOVE ME)")
+        shutil.rmtree('llvm.obj.1', ignore_errors=True)
+        shutil.copytree(os.path.join('..', 'llvm.obj.1'), 'llvm.obj.1')
```

Comment:
@Crivella This looks like some testing stuff that needs to be cleaned up?

Reply:
Yeah sorry ended up committing it by mistake.
Removed the comments and kept it as a stash in case i will still need them.
It was used to copy previous stage 1 and 2 builds when i was doing changes only to stage 3 to speed up the builds